orttraining/orttraining/models/bert/main.cc

```diff
@@ -768,6 +768,8 @@ static Status RunTraining(const BertParameters& params, const Environment& env)
 }
 
 int main(int argc, char* argv[]) {
+  bool keep_running = true;
+  while(keep_running);
```

Comment:
clean up?

Reply:
agreed, will remove before check-in.

---
In reply to: [453882253](https://github.com/microsoft/onnxruntime/pull/4443#discussion_r453882253) [](ancestors = 453882253)